frontend/src/queries/schema.ts

```diff
@@ -357,8 +378,6 @@ export interface RecentPerformancePageViewNode extends DataNode {
     numberOfDays?: number // defaults to 7
 }
 
-export type HogQLExpression = string
```

Comment:
Did this not work with the type generator? In general I think it's nice to differentiate HogQL expressions from regular strings.

Reply:
It basically nested it in 
```py
class HogQLExpressoin
    __root__: str
```

so I'd have to do `expr.__root__` to get the value.

I ran into a similar problem with `AnyPropertyFilters` in `query_event_list.py`. There's likely a cleaner solution... hence why it's still draft :).

Reply:
I found an option `datamodel-codegen --collapse-root-models` that collapses all types that have just one `__root__` inside them. This way we can continue using `HogQLExpressions`-s in the frontend.